src/collections/_documentation/error-reporting/configuration/drain-example/native.md

```diff
@@ -6,3 +6,6 @@ sentry_shutdown();
 
 Calling `sentry_shutdown()` before exiting the application is critical to avoid
 data loss.
+
+After shutdown the client cannot be used any more so make sure to only do that
+right before you shut down the application.
```

Comment:
```suggestion
 before you shut down the application.
```

Reply:
Thinking about "right before" vs "before".

In this case, "before" sounds like any time before -- not so precise.
Conversely, I think the intention of "right before" was to mean "call `close` and the very next thing is the end of the app". The latter is the correct, precise suggestion.

@MimiDumpling could you elaborate on why we would remove "right"?

Reply:
Made a suggestion in the comment above. 😄 

The word "right" isn't quite precise enough in this instance. "Right" has many meanings. I think a better word for this case is "immediately."